src/plugins/better-navigation/index.less

```diff
@@ -0,0 +1,6 @@
+body.dsm-better-nav-scrollable-expressions {
+  .dcg-mq-root-block.dcg-mq-root-block.dcg-mq-root-block.dcg-mq-root-block.dcg-mq-root-block.dcg-mq-root-block.dcg-mq-root-block.dcg-mq-root-block.dcg-mq-root-block {
```

Comment:
average !important hater :)

Reply:
I find it funny that out of the two ways of improving CSS specificity without easy control over the HTML, this is generally the _better_ one.